corehq/apps/translations/app_translations/utils.py

```diff
@@ -202,6 +205,49 @@ def none_or_unicode(val):
     return rows
 
 
+def text_and_audio_changed(current_row, previous_row, headers):
+    text_changed = False
+    audio_path_changed = False
+    for index, header in enumerate(headers):
+        if "default_" in header and previous_row[index] != current_row[header]:
+            text_changed = True
+        if "audio_" in header and previous_row[index] != current_row[header]:
+            audio_path_changed = True
```

Comment:
nit:
may be can exit early if both are already true
```
if text_changed and audio_path_changed:
    break
```

Reply:
Functionality moved to icds repo -  https://github.com/dimagi/commcare-icds/pull/147/files#diff-b09d6202e8e919202144f99ff097824531989b6b9f0524a5439d05026e22ae32R49-R50